### PR TITLE
[desktop] Show notification after autosaving

### PR DESCRIPTION
**Pull request by twiss**
To make it clear that autosave exists.

Also, make notifications more prominent.

### CODE REVIEW

File: scripts/notifications.js [hunk 1]
```diff
@@ -27,6 +27,13 @@ firetext.notify = function (message, title, time) {
 	notification.appendChild(notificationBody);
 	
 	document.body.appendChild(notification);
+	
+	if (deviceType == 'desktop') {
+		notification.style.left = Math.floor((window.innerWidth - notification.offsetWidth) / 2) + 'px';
```
Comment (ferndot) [inline scripts/notifications.js]: This is nice, but not responsive. Can you try using `calc` in the css instead?

**Comment by ferndot** [comment]
Thanks @twiss! I think we can merge once the responsiveness issue is fixed :)

**Comment by twiss** [comment]
Fixed

**Comment by ferndot** [comment]
:+1: ready for merge!

**Pull request merged**